frontend/amundsen_application/static/js/features/NavBar/index.tsx

```diff
@@ -245,6 +299,24 @@ export const NavBar: React.FC<NavBarProps> = ({ loggedInUser, location }) => {
     setRunTour(false);
   }, [pathname]);
 
+  const handleAppSuiteToggle = (isOpen: boolean) => {
+    logAction({
+      target_id: '',
+      command: 'click',
+      target_type: 'button',
+      label: isOpen ? 'Open App Suite Menu' : 'Close App Suite Menu',
```

Comment:
can we make these strings into constants?

Reply:
I usually don't do it for tracking messages, but yes, I can do that.